README.md

```diff
@@ -625,6 +733,14 @@ Future<void>.delayed(Duration.zero, () {
 });
 ```
 
+## Support, feedback and troubleshooting
+
+Please visit <http://support.ably.com/> for access to our knowledge base and to ask for any assistance.
```

Comment:
I can see this has been inherited from [our `ably-java` readme](https://github.com/ably/ably-java/#support-feedback-and-troubleshooting), however we should be promoting `https`, and actually this now resolves to `https://ably.com/support`.

```suggestion
Please visit [ably.com/support](https://ably.com/support) for access to our knowledge base and to ask for any assistance.
```

Reply:
Fixed in https://github.com/ably/ably-flutter/pull/413/commits/112d5c75a6babeb050304a3ed21b2bddd4773b4f